src/components/combo/types.ts

```diff
@@ -35,10 +36,10 @@ export interface GroupingOptions<T extends object> {
 }
 
 /* marshalByValue */
-export interface IgcComboChangeEventArgs {
-  newValue: string[];
+export interface IgcComboChangeEventArgs<T extends object = any> {
+  newValue: ComboValue<T>;
   /* primitiveValue */
```

Comment:
@gmurray81 This I'm assuming is likely either wrong or not correct anymore? Items were not primitive values even before, they are actual data items, always. Which would mean they'd likely be tracked and this can be dropped?

Reply:
I think we might be treating a ref to an item as a primitive value. I'd have to check.